samples/003-SuspensionMng/README.md

```diff
@@ -0,0 +1,40 @@
+# getUsers
+
+## Description
+
+This plugin is designed to accumulate the duration of all the suspended segments of one activity.
+It also is able to count the time between the first start time and the final completion time.
+
+## Parameters
+
+**Secure Parameters**
+
+None
+
+**Open Parameters**
+
+- `openAction`: Action that indicates if we are suspending or completing the activity. Possible values : [`suspend`, `complete suspended`]
+
+**Properties needed**
+
+None
```

Comment:
Can you please double check this? I thought the values were using some custom properties

Reply:
According to the descriptor:
           "appt_number",
           "length",
           "ETA",
           "astatus",
           "aid",
           "date",
           "A_TOTAL_DURATION",
           "A_TOTAL_DURATION_COMPLETE",
           "A_ORIGINAL_START_TIME",
           "A_ORIGINAL_START_TIME_TZ",
           "A_TOTAL_TIME_TO_SOLVE"